File: VB/WebSite/clsNames.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text

Public Class clsNames
	Inherits List(Of clsName)
	Public Sub New()
		Me.Add(New clsName("A", "Arnold", "Apple"))
		Me.Add(New clsName("B", "Barney", "Bagle"))
		Me.Add(New clsName("C", "Charlie", "Chipman"))
		Me.Add(New clsName("D", "Doug", "Doggle"))
		Me.Add(New clsName("E", "Elizabeth", "Einstein"))
		Me.Add(New clsName("F", "Frank", "Flanery"))
		Me.Add(New clsName("G", "Gilbert", "Guzman"))
		Me.Add(New clsName("H", "Hillary", "Harvey"))
		Me.Add(New clsName("I", "Isadora", "Ingleman"))
		Me.Add(New clsName("J", "Jacob", "James"))
		Me.Add(New clsName("K", "Kevin", "Krantz"))
	End Sub
End Class

File: CS/WebSite/clsNames.cs
using System;
using System.Collections.Generic;
using System.Text;

public class clsNames : List<clsName>
{
    public clsNames()
    {
        this.Add(new clsName("A", "Arnold", "Apple"));
        this.Add(new clsName("B", "Barney", "Bagle"));
        this.Add(new clsName("C", "Charlie", "Chipman"));
        this.Add(new clsName("D", "Doug", "Doggle"));
        this.Add(new clsName("E", "Elizabeth", "Einstein"));
        this.Add(new clsName("F", "Frank", "Flanery"));
        this.Add(new clsName("G", "Gilbert", "Guzman"));
        this.Add(new clsName("H", "Hillary", "Harvey"));
        this.Add(new clsName("I", "Isadora", "Ingleman"));
        this.Add(new clsName("J", "Jacob", "James"));
        this.Add(new clsName("K", "Kevin", "Krantz"));
    }
}
